src/pages/Viewer.js

```diff
@@ -50,16 +50,55 @@ function Viewer({ code, widgets }) {
     fetchRedirectMap();
   }, []);
 
+  const routes = {
+    home: {
+      path: "buildhub.near/widget/page.home",
+      blockHeight: "final",
+      init: {
+        name: "Home",
+      },
+    },
+    feed: {
```

Comment:
We need to be conscience of the different environments; see data/widgets.js for swapping between mainnet and testnet